haas/cli.py

```diff
@@ -401,6 +431,24 @@ def stop_console(node):
     do_delete(url)
 
 @cmd
+def make_initial_admin(username, password):
```

Comment:
What do you think of naming this `create_admin_user`?

We use "create" elsewhere and there's no restriction (other than our suggestion/good admin practices) that it only be run initially.